docs/BackupAndRestore.md

```diff
@@ -103,6 +103,42 @@ Use `nuobackup --type report-latest --db-name <db> --group <backup group>` to sh
 It is possible to disable backup jobs creation by setting `hotcopy.enableBackups` to `false`.
 Custom backup jobs that use the `nuobackup` script can be configured to support more complex workflows and backup strategies.
 
+The logs for each backup job can be seen using `kubectl get logs <job pod name>`. For example:
+
+```bash
+kubectl logs full-hotcopy-demo-cronjob-1621415040-vxlq8 --namespace nuodb
```

Comment:
Just to be consistent with other example commands remove "--namespace nuodb"

Reply:
throughout the docs, examples and install guides we always use `-n nuodb` or `--namespace nuodb`. Rather than removing it here, I would sugest adding it where it is missing :)

Reply:
I will make sure that `--namespace nuodb` is everywhere in the guide.